examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -0,0 +1,19 @@
+{
+    "include": "../../../../src/app/tests/suites/ciTests.json",
+    "disable": [
```

Comment:
The .js version includes comments explaining _why_ things are disabled.  We need to keep that information somehow.  Perhaps what we need is an array of objects each of which has the test to disable and a reason why.